scripts/rtma3d/exrtma3d_rhist.ksh

```diff
@@ -131,75 +131,66 @@ ln -sf  $COMOUThrrrdas_rtma3dm2 $DATA/$PDYm2
 ln -sf  $COMOUThrrrdas_rtma3dm3 $DATA/$PDYm3
 
 
-ls -1r */*/* | awk '
-
-            /prepbufr/ { print "./"$0 > "perm" ; next }
-
-	    /bufr/ { print "./"$0 > "perm" ; next }
+ls -1r */* | awk '
 
-            /grib2/ { print "./"$0 > "perm" ; next }
+            /rejectlist/ { print "./"$0 > "perm1" ; next }
 
-	    /grb2/ { print "./"$0 > "perm" ; next }
+            /smarthrrrconus/ { print "./"$0 > "perm1" ; next }
 
-            /rejectlist/ { print "./"$0 > "perm" ; next }
+	    /smarthrrrak/ { print "./"$0 > "perm1" ; next }
```

Comment:
Please search for /smart/ instead as smarthrrrak is not the naming convention used.